### PR TITLE
JSWeek3 Homework

### PR DESCRIPTION
**Pull request by CharlesKimMaina**

### CODE REVIEW

File: javascript/javascript3/week1/cityWeatherApi.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        .then(response => response.json())
+        .then(data => {
+            var nameValue = data["name"];
+            var temperatureValue = data["main"]["temp"];
+            var windSpeedValue = data["wind"]["speed"];
+            var cloudyStatusValue = data["weather"][0]["description"];
+            var sunriseValue = data["sys"][2]||[3];
+            
+ 
+            name.innerHTML = nameValue;
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/cityWeatherApi.html:78]: `name` doesn't render anything. Try to give it a better name and see what happens

File: javascript/javascript3/week1/cityWeatherApi.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+</style>
+<body>
+
+    <div class="input">
+        <input type="text" class="inputValue" placeholder="Enter a city"required>
+        <input type="submit" value="Submit" class="button">
+    </div>
+
+    <div class="display">
+        <h1 class="name"></h1>
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/cityWeatherApi.html:47]: `name` of what?

File: javascript/javascript3/week1/cityWeatherApi.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        var sunrise = document.querySelector(".sunrise")
+        
+        
+
+        button.addEventListener("click", function(){
+            fetch('http://api.openweathermap.org/data/2.5/weather?q='+inputValue.value+'&APPID=83213d88197c73f0c991024226083a52')
+        .then(response => response.json())
+        .then(data => {
+            var nameValue = data["name"];
+            var temperatureValue = data["main"]["temp"];
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/cityWeatherApi.html:72]: you can also access the same data as `data.main.temp`

File: javascript/javascript3/week1/gifAPI.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+</style>
+
+<body>
+ <h1>Giffy Searcher</h1>
+
+ <div id="container-input">
+    <input type="text" id="input-textbox" placeholder="Search for gifs" required><br>
+    <button type="button" id="btn">Search for gifs here!</button><br>
+    <input type="text" id="input-number" placeholder="Number of gifs" required>
+    <ul id =gifImage></ul>
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/gifAPI.html:45]: you forgot quotes in `gifImage`, and why is the name in camelCase? Use kebab-case, as for other classes and ids

File: javascript/javascript3/week1/cityWeatherApi.html (hunk 2 above)
Comment (CharlesKimMaina) [reply javascript/javascript3/week1/cityWeatherApi.html:47]: city name

File: javascript/javascript3/week1/cityWeatherApi.html (hunk 3 above)
Comment (CharlesKimMaina) [reply javascript/javascript3/week1/cityWeatherApi.html:72]: yes, thanks,.,i have changed according to your suggestions,.,

**Pull request merged**